R/calc_views.R

```diff
@@ -80,8 +80,8 @@ calc_view_usr <- function(views){
     
     if ((side %% 2) == 0){
       # is y 
-      if(length(grep("^y$", unlist(lapply(side_components, names)))) > 0){
-        lims <- lims_from_list(lapply(side_components, var='y', function(list, var) strip_pts(list,var)))
+      if (any(unname(unlist(lapply(side_components, names))) %in% c('y','y1','y0'))){
+        lims <- lims_from_list(lapply(side_components, var=c('y','y1','y0'), function(list, var) strip_pts(list,var)))
```

Comment:
@ldecicco-USGS do you have a minute to talk about these other places, i'm not totally sure I (we, maybe @lindsaycarr wants to join us too) understand what is going on with the code enough to edit it. :/